lib/controller/WebPushController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace CsrDelft\controller;
+
+use CsrDelft\common\Annotation\Auth;
+use CsrDelft\repository\WebPushRepository;
+use CsrDelft\service\security\LoginService;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+class WebPushController extends AbstractController
+{
+	/**
+	 * @param WebPushRepository $webPushRepository
+	 * @param Request $request
+	 * @return JsonResponse
+	 * @Route("/webpush-subscription", methods={"POST", "PUT", "DELETE"})
```

Comment:
Je kan verschillende functies maken voor verschillende methods, en deze wel dezelfde route geven, maakt het wat overzichtelijker.

Reply:
Ah ja, ik wist al niet zeker of er een betere manier was, maar dit klinkt wel beter ja